modules/op-nonroot-buildah-user-namespaces.adoc

```diff
@@ -0,0 +1,70 @@
+// This module is included in the following assemblies:
+// * secure/unprivileged-building-of-container-images-using-buildah.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="nonroot-buildah-user-namespaces_{context}"]
+= Running Buildah as a non-root user by configuring user namespaces
+
+Configuring user namespaces is the simplest way to run Buildah in a task as a non-root user. However, some images might not build using this option.
+
+.Prerequisites
+
+* You have installed the `oc` command-line utility.
+
+.Procedure
+
+. Create a copy of the `buildah` task, which is provided in the `openshift-pipelines` namespace; change the name of the copy to `buildah-as-user`.
```

Comment:
a rewording suggestion, if the user is supposed to both copy and rename the copy with the listed command:
```suggestion
. Create a copy of the `buildah` task, which is provided in the `openshift-pipelines` namespace, and change the name of the copy to `buildah-as-user` with the following command:
```